### ISSUE TITLE
Spotting disease disrupts the microbiome of infected purple sea urchins, Strongylocentrotus purpuratus 

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#311]
Spotting disease disrupts the microbiome of infected purple sea urchins, Strongylocentrotus purpuratus – Chloe G. Shaw – BMC Microbiology
https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-023-03161-9

**Comment by Druscillaa** [comment]
Hey @SvetlanaUP can I please be assigned this paper

**Comment by Druscillaa** [comment]
Hello @SvetlanaUP please review my study

https://bugsigdb.org/Study_1018

**Comment by Scholarpat** [comment]
Hi @SvetlanaUP,

Here are the changes I made for Experiment 1 of this study:

- Included data transformation.
- Removed ANOVA and retained only LEfSe for the statistical test.
- Added multiple hypothesis testing (MHT) correction.
- Changed the alpha diversity indexes from decreased to unchanged.

I created four additional experiments with signatures that are not displaying: signatures 1 and 2 for Experiment 2, and signature 1 for Experiments 3, 4, and 5.

This curation is now complete. Thank you!
